### PR TITLE
T9997: Replace Miraheze-specific messages with generic ones

### PR DESCRIPTION
**Pull request by redbluegreenhat**
This pull request replaces some of the default interface messages with more generic ones that don't link to Miraheze-specific pages or roles such as Wiki Creators.

To prevent disruption to Miraheze, this PR shouldn't be merged until these messages are added as overrides in Miraheze's MW configuration.

### CODE REVIEW

**Comment by redbluegreenhat** [comment]
MIrahezeMagic PR: https://github.com/miraheze/MirahezeMagic/pull/385

File: i18n/en.json [hunk 1]
```diff
@@ -55,17 +56,17 @@
...
+	"requestwiki-error-invalidcomment": "Your wiki request contains an invalid character. Please try requesting again.",
+	"requestwiki-error-patient": "You have already requested a wiki. Please be patient and we should create it soon. If you need to request another wiki, please wait and you will be able to request later.",
+	"requestwiki-header": "Thank you for choosing us to host your wiki! To get started, please fill out this form so we can review your request.\nYour wiki request will be reviewed and we will let you know if we decline or approve this request.",
 	"requestwiki-label-bio": "Will this wiki's main subject(s) be about living people?",
 	"requestwiki-label-language": "Language:",
 	"requestwiki-label-private": "Private wiki (not readable by everyone; only readable by users given the member or sysop groups):",
 	"requestwiki-label-purpose": "The purpose of this wiki is...",
 	"requestwiki-label-sitename": "Wiki name:",
-	"requestwiki-label-siteurl": "Subdomain: (subdomain.miraheze.org)",
+	"requestwiki-label-siteurl": "Subdomain:",
```
Comment (Universal-Omega) [inline i18n/en.json:67]: This one can stay, I will make the domain a $1 variable based on configuration later.

**Review by Universal-Omega — CHANGES_REQUESTED** [review]

File: i18n/en.json (hunk 1 above)
Comment (redbluegreenhat) [reply i18n/en.json]: Done.

**Comment by Universal-Omega** [comment]
Merge conflict (sorry about that)

**Comment by Universal-Omega** [comment]
#360 does the last message for this also by the way.

**Comment by Universal-Omega** [comment]
Also some things in qqq.json need updated (like the doc things saying not to translate "Miraheze" removed, since that is not in messages anymore), and probably those doc messages added to MirahezeMagic overrides qqq.json.

**Comment by redbluegreenhat** [comment]
> Merge conflict (sorry about that)
conflict dealt with

**Comment by redbluegreenhat** [comment]
> Also some things in qqq.json need updated (like the doc things saying not to translate "Miraheze" removed, since that is not in messages anymore), and probably those doc messages added to MirahezeMagic overrides qqq.json.

I tried, didn't know what to put in createwiki-close-email-sender, so I just left it as a empty string.

**Comment by Universal-Omega** [comment]
> > Also some things in qqq.json need updated (like the doc things saying not to translate "Miraheze" removed, since that is not in messages anymore), and probably those doc messages added to MirahezeMagic overrides qqq.json.
> 
> I tried, didn't know what to put in createwiki-close-email-sender, so I just left it as a empty string.

Just put noreply there.

**Comment by redbluegreenhat** [comment]
> The default we use here, could also then just be the default messages from MediaWiki itself, and remove the message entirely, besides the MirahezeMagic one that would now not be an override but a config of a base message.
On it. I'll remove the main page message here and move the one at the MirahezeMagic PR to en.json.
> Additional, regarding the MirahezeMagic PR, there are few options to handle the fact that all these messages will need regenerated. Especially for main page message.
> 
>     1. Don't add the overrides in hooks, keep them just in i18n file until they rebuild before doing this PR, so both instances exist, using the current CW messages until they are rebuilt.
> 
>     2. Copy over all languages of the messages to MirahezeMagic (they don't all need changed here).
> 
>     3. See if TranslateWiki can assist with copying everything over.
I don't really know what to say. Given the third option, you're concerned with losing the translations?

**Comment by Universal-Omega** [comment]
Yes, in particular the main page, because of how large the message is it won't be translated again bery fast. I'll talk with someone from translatewiki and see if they can import/move them.

**Comment by redbluegreenhat** [comment]
Main page message removed here.

File: i18n/en.json [hunk 2]
```diff
@@ -2,18 +2,18 @@
...
...as needed. To start, try checking out these helpful links:\n* [[mw:Special:MyLanguage/Help:Contents|MediaWiki guide]] (e.g. navigation, editing, deleting pages, blocking users)\n* [[meta:Special:MyLanguage/FAQ|Miraheze FAQ]]\n* [[meta:Special:MyLanguage/Request features|Request settings changes on your wiki]]. (Extensions, Skin and Logo/Favicon changes should be done through [[Special:ManageWiki]] on your wiki, see [[meta:Special:MyLanguage/ManageWiki|ManageWiki]] for more information.)\n\n==== I still don't understand X! ====\nWell, that's no problem. Even if something isn't explained in the documentation/FAQ, we are still happy to help you. You can find us here:\n* [[meta:Special:MyLanguage/Help center|On our own Miraheze wiki]]\n* On [[phab:|Phabricator]]\n* On [https://miraheze.org/discord Discord]\n* On IRC in #miraheze on irc.libera.chat ([irc://irc.libera.chat/%23miraheze direct link]; [https://web.libera.chat/?channel=#miraheze webchat])\n\n=== For visitors of this wiki ===\nHello, the default Main Page of this wiki (this page) has not yet been replaced by the bureaucrat(s) of this wiki. The bureaucrat(s) might still be working on a Main Page, so please check again later!",
```
Comment (Universal-Omega) [inline i18n/en.json:11]: This can stay here for now. Changing if can be a follow-up due to it being a lot larger.

**Review by Universal-Omega — CHANGES_REQUESTED** [review]

File: i18n/en.json (hunk 2 above)
Comment (redbluegreenhat) [reply i18n/en.json:11]: Alright.

**Comment by redbluegreenhat** [comment]
This PR had a merge conflict, I solved it now.

**Pull request closed without merging**